### DISCUSSION TITLE
AWS Migration?

### DISCUSSION THREAD

**Discussion by lhwostow001** [akash-network/community#490] [Q&A]
Hello. I have a small web application hosted on AWS that I'd like to run on Akash. However, I'm not sure it is worth the lift and shift effort or I should just start new in Akash. If you are interested in having a short discussion to help me think this through let me know.

**Comment by rodri-r** [comment]
Hey, in my  opinion it is definitely worth the lift and effort to move your app to Akash. You will get quicker help if you join the official Akash Discord and ask this in there.

**Reply by ghost** [reply]
Adding Akash Discord link https://discord.gg/akash

**Comment by anilmurty** [comment]
Will have to understand the specifics better. But in general, if you are running dockerized applications orchestrated using something like EKS - then lift and shift would make complete sense. You would need to port any docker compose type files to Akash SDL and take it from there. If you are not running containeraized/ orchestrated apps in AWS then a start from fresh might make sense. That's just general advice ofcourse and as the folks above suggested, best to take this to discord
